src/pepr/operator/controllers/network/authorizationPolicies.ts

```diff
@@ -261,6 +263,25 @@ export async function generateAuthorizationPolicies(
     }
   }
 
+  // In Sidecar mode, Prometheus lacks mTLS credentials, causing metric scrapes on port 15020 to fail.
+  // Add an AuthorizationPolicy to allow all traffic on port 15020 for the package's namespace.
```

Comment:
`Prometheus lacks mTLS credentials` is potentially the opposite of the actual issue here. I don't know that I can explain it well, but Prometheus _does_ have mTLS credentials/identity (otherwise you couldn't use the principal in this ALLOW policy).

I believe the issue is essentially that when Prometheus is in ambient mode, ALL traffic goes through the ztunnel proxy, and subsequently gets evaluated by the destination sidecar and blocked without a matching authpol. Previously in sidecar mode Prometheus didn't send any traffic through its sidecar (technically) and the destination accepted it as plaintext because it was the sidecar metrics rather than app metrics.

```suggestion
  // With Prometheus in Ambient mode, all traffic is sent over mTLS and the destination sidecar requires an ALLOW policy to expose sidecar metrics.
  // Add an AuthorizationPolicy to allow all traffic on port 15020 for the package's namespace.
```

Feel free to adjust wording as you see fit.